Game/Levels/EqualityWorld/L04_UnfoldingDefinitions.lean

```diff
@@ -1,5 +1,5 @@
 import Game.Metadata
-import Game.Levels.FunctionWorld.L07_SwappingInputs
+import Game.Levels.FunctionWorld
```

Comment:
You should generally only import entire worlds within another world, not individual levels. There is a known bug here that the inventory will behave weirdly.

Importing levels of the same world is fine